spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe VideosController do
+  describe "GET show" do
+    context "with authenticated users" do
+      before do
+        session[:user_id] = Fabricate(:user).id #to log us in for running the test
+      end
+
+      it "sets @reviews for authenticated users" do
```

Comment:
`for authenticated users` this part of the example message is redundant because you have already set the context.